hacks_mbof/settings.py

```diff
@@ -186,3 +186,83 @@
         },
     },
 }
+
+#Shib
+
+SAML2_URL_PATH = '/accounts/'
+# modify to use port request comes
+SAML2_URL_BASE = getenv('DJANGO_SAML2_URL_BASE', 'http://localhost:18000/accounts/')
+
+INSTALLED_APPS += ('djangosaml2',)
+AUTHENTICATION_BACKENDS = (
+    'django.contrib.auth.backends.ModelBackend',
+    'djangosaml2.backends.Saml2Backend',
+)
+LOGIN_URL = '%slogin/' % SAML2_URL_PATH
```

Comment:
For a simple string appending, use `SAML2_URL_PATH + '/login/'` instead.  Putting an extra slash before `login` won't hurt and you won't need to worry about whether the value of `SAML2_URL_PATH` ended with one.